main.tex

```diff
@@ -50,99 +61,204 @@
 
 \subsubsection{Block-space efficiency}
 
 The rigidity of the current transaction structure, i.e. fixed denominations, constrains users' unspent transaction output set structure as well. These limitations force users to consolidate their coins (see \cref{fig:postmixmerging}) and create additional intermediate outputs with constrained amounts when interspersing CoinJoin transactions with transactions that send or receive value.
 
-\begin{figure}[h!]
+\begin{figure}%[ht!]
+\centering
+\begin{minipage}{.45\textwidth}
+    \centering
+    \includegraphics[width=\linewidth]{Figures/avgRemixCount.pdf}
+    \caption{Average remix count of various Bitcoin privacy-enhancing tools. Remixing increases user privacy at the expense of block-space efficiency. }
+    \label{fig:remixcount}
+\end{minipage}
+\hspace{6pt}
+\begin{minipage}{.45\textwidth}
     \centering
-    \includegraphics[scale=0.4]{Figures/postMixInputMerging.pdf}
-    \caption[]{Average number of inputs from the first post-mix transactions in various CoinJoin schemes.\footnote{The large figures for other CoinJoin-like transactions suggests false positives in our identification.}}
+    \includegraphics[width=0.95\linewidth]{Figures/postMixInputMerging.pdf}
+    \caption{Average number of inputs from the first post-mix transactions in various CoinJoin schemes. The large figures for other CoinJoin-like transactions suggests false positives in our identification.}
     \label{fig:postmixmerging}
+\end{minipage}
 \end{figure}
 
-\subsubsection{Lack of privacy-enhanced payments} Currently Wasabi supports neither payments from a CoinJoin, nor payments in a CoinJoin. Payments from a CoinJoin would protect sender privacy and improve efficiency by requiring fewer intermediate outputs. Payments within a CoinJoin would protect both sender and receiver privacy, and since they are a form of PayJoin\footnote{\url{https://en.bitcoin.it/wiki/PayJoin}} it would also improve privacy by introducing degrees of freedom in the interpretation of CoinJoins.
-
-\subsection{Our Contribution}
+\subsubsection{Lack of privacy-enhanced payments} Currently, Wasabi supports neither payments from a CoinJoin, nor payments in a CoinJoin. Payments from a CoinJoin would protect sender privacy and improve efficiency by requiring fewer intermediate outputs. Payments within a CoinJoin would protect both sender and receiver privacy, as well as undermine clustering heuristics since they are analogous to PayJoins. It would also improve privacy by introducing degrees of freedom in the interpretation of CoinJoins.
```

Comment:
@nothingmuch you are the first I know to say `payment from CoinJoin` and `payment within CoinJoin`. 
is this sufficiently explained here?

Reply:
this is explained in more detail in the examples section, i remember working on this paragraph and giving up trying to explain the difference precisely before the concept of credentials was introduced